src/filters.c

```diff
@@ -399,6 +399,32 @@ const struct pl_filter_function pl_filter_function_gaussian = {
     .tunable   = {true},
 };
 
+static double power_of_blackman(const struct pl_filter_ctx *f, double x)
+{
+    return pow(blackman(f, x), f->params[1]);
+}
+
+const struct pl_filter_function pl_filter_function_power_of_blackman = {
+    .name    = "power_of_blackman",
+    .weight  = power_of_blackman,
+    .radius  = 1.0,
+    .params  = {0.16, 1.0},
+    .tunable = {true, true},
+};
+
+static double power_of_garamond(const struct pl_filter_ctx *f, double x)
+{
+    return pow(1.0 - pow(x, f->params[0]), f->params[1]);
+}
+
+const struct pl_filter_function pl_filter_function_power_of_garamond = {
+    .name    = "power_of_garamond",
```

Comment:
This name breaks convention a lot. Just call it "garamond", or maybe just "power" to be more descriptive of what it actually does.

Reply:
Changing name is fine by me. I named the original window `1.0 - pow(x, n)` Garamond, since I needed some name for it and later this one was named Power of Garamond. Same as with Power of Blackman change, I would like to indicate that is Power of Garamond at least leave comment in the code.